worldedit-fabric/src/main/java/com/sk89q/worldedit/fabric/FabricWorld.java

```diff
@@ -166,8 +168,44 @@ public Path getStoragePath() {
         return null;
     }
 
+    public void markAndNotifyBlock(World world, BlockPos pos, @Nullable WorldChunk worldChunk, net.minecraft.block.BlockState blockState,
```

Comment:
I wonder if mixins has some trick we can use to call this dynamically, rather than needing to copy it out.

Reply:
Copying it out makes it easier to understand IMO, also IIRC this didn't actually exist within Fabric, it's embedded within a method. Forge & Bukkit both split it out to its own method.